src/vs/workbench/contrib/chat/browser/chatAccessibilityService.ts

```diff
@@ -16,34 +16,34 @@ export class ChatAccessibilityService extends Disposable implements IChatAccessi
 
 	declare readonly _serviceBrand: undefined;
 
-	private _responsePendingAudioCue: IDisposable | undefined;
-	private _hasReceivedRequest: boolean = false;
+	private _pendingCueMap: Map<number, IDisposable | undefined> = new Map();
+
 	private _runOnceScheduler: RunOnceScheduler;
 	private _lastResponse: string | undefined;
+	private _requestId: number = 0;
 
 	constructor(@IAudioCueService private readonly _audioCueService: IAudioCueService) {
 		super();
 		this._register(this._runOnceScheduler = new RunOnceScheduler(() => {
```

Comment:
If you schedule this twice in a row quickly, I think it will only run until the second request is done? In reality I think you will rarely have two requests going simultaneously.